packages/ra-ui-materialui/src/input/ResetableTextField.js

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from 'react';
```

Comment:
This should be the default TextField IMO. Make resetable a prop

Reply:
You mean all components using the mui TextField should use this one ?